tools/appscript/lib/schedule.mjs

```diff
@@ -489,8 +497,8 @@ function addSessions(sheet, project, validationErrors) {
         }
         first = false;
         tokens.push({
-          label: '#' + number,
-          href: `${sessionsSheetUrl}&range=${findSessionRange(project, number)}`
+	  ...(true) && {label: '#' + number, href: `${sessionsSheetUrl}&range=${findSessionRange(project, number)}`},
+	  ...conflictIssues.includes(number) && {style: conflictHighlight}
```

Comment:
Interesting syntax ;)

Once you're within `{`, you're dealing with an object, unless I missed something, the spread syntax ('...') is not going to work. More importantly, the spread syntax applies to arrays to create a list of elements. Here you give it an object.

If you're willing to avoid `if` clauses for some reason, extending an object with another one is typically done through `Object.assign()`.

All in all, you'll need something such as:

```js
tokens.push(Object.assign(
  { label: 'coucou' },
  conflictIssues.includes(number) && { style: conflictHighlight }
));

... but `conflictIssues` does not seem to be an array of error numbers, rather an array of errors, so `includes` is not going to work either.